environment.yml

```
@@ -11,14 +11,14 @@ dependencies:
   - gdal>=3
```

```suggestion
  - gdal>=3.6
```

Should we specify `gdal>=3.6` here now since that seems to be the de facto minimum supported version?

https://github.com/isce-framework/isce3/blob/3040629593a08a536e57923204251f8407501787/.github/conda_envs/minimum-reqs.yml#L11-L15